Dashboard/ViewModels/FooterViewModel.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Globalization;
+using System.Timers;
+using Dashboard.Connectors;
+using Dashboard.Models;
+
+namespace Dashboard.ViewModels;
+
+public partial class FooterViewModel : ViewModelBase
+{
+    public bool? HeartBeat => _dataStore.HeartBeat;
+    public event Action<bool>? HeartbeatStatusUpdated;
+    public event Action<string>? RawMessageUpdated;
+    public event Action<bool>? ConnectionUpdate;
+    private readonly IDataStore _dataStore;
+    public bool ConnectionStatus => _dataStore.RawData?.ConnectionStatus ?? false;
+    public string CarId => _dataStore.RawData?.CarId ?? "0";
+    public string RawMessage => _dataStore.RawData?.RawMessage ?? "";
+    public string UTcTime
+    {
+        // Convert the UTC value from data logger to proper UTC.
+        get
+        {
+            var utcTimeString = _dataStore.RawData?.UTCTime;
+            if (string.IsNullOrEmpty(utcTimeString))
+            {
+                return "Invalid time";
+            }
+            try
```

Comment:
additional q: do we even need to pull this from the av logger? can we just do this ourselves and get the current time in C#?

Reply:
We could do this at the serial layer so other pages could use the times if needed.

As for not pulling it from the logger, I think it would depend on if we're making this in prep for replaying logs and if Nick O wants this UTC time to be the current or time of the original log.

Reply:
Discussed in the meeting with nick yesterday, will use the time from the logger as it will allow more flexibility when we add replay from file